pandora.py

```diff
@@ -10,6 +10,10 @@ class AudioFormat(enum.Enum):
     MP3_HIFI = 'mp3-hifi'
     AACPLUS = 'aacplus'
 
+class NewStationType(enum.IntEnum):
```

Comment:
You aren't actually using this value as an `int` so it might as well be `enum.Enum`.

Reply:
Whatever you like. 6 of one thing 1/2 dozen of another. I'll change it.